Formula/j/jj.rb

```diff
@@ -26,7 +26,13 @@ def install
 
     system "cargo", "install", *std_cargo_args(path: "cli")
 
-    generate_completions_from_executable(bin/"jj", "util", "completion", shell_parameter_format: :flag)
+    generate_completions_from_executable("env", "COMPLETE=bash", bin/"jj",
+      shells: [:bash], shell_parameter_format: :none)
+    generate_completions_from_executable("env", "COMPLETE=fish", bin/"jj",
+      shells: [:fish], shell_parameter_format: :none)
+    generate_completions_from_executable("env", "COMPLETE=zsh", bin/"jj",
+      shells: [:zsh], shell_parameter_format: :none)
```

Comment:
What checks for the `COMPLETE` variable? Is it `jj` itself, or some crate that they're using?

Reply:
It’s a feature of the clap (clap_complete to be specific) crate: https://docs.rs/clap_complete/latest/clap_complete/env/index.html

Reply:
Maybe, we have to add `clap` style for the `shell_parameter_format` from `homebrew/brew` to handle it as general format.

Reply:
Now this should work.
```suggestion
    generate_completions_from_executable(bin/"jj", shell_parameter_format: :clap)
```